File: code/Dpr.py
class Dpr:

    __idd = ""
    __nama = ""
    __nama_bidang = ""
    __nama_partai = ""

    def __init__(self, idd="", nama="", nama_bidang="", nama_partai=""):
        self.__idd = idd
        self.__nama = nama
        self.__nama_bidang = nama_bidang
        self.__nama_partai = nama_partai

    def get_idd(self):
        return self.__idd

    def set_idd(self, idd):
        self.__idd = idd

    def get_nama(self):
        return self.__nama

    def set_nama(self, nama):
        self.__nama = nama

    def get_nama_bidang(self):
        return self.__nama_bidang

    def set_nama_bidang(self, nama_bidang):
        self.__nama_bidang = nama_bidang

    def get_nama_partai(self):
        return self.__nama_partai

    def set_nama_partai(self, nama_partai):
        self.__nama_partai = nama_partai

File: code/Dpr.java
public class Dpr
{
    private String id, nama, nama_bidang, nama_partai;


    public Dpr()
    {
        this.id = "";
        this.nama = "";
        this.nama_bidang = "";
        this.nama_partai = "";
    }

    public Dpr(String id, String nama, String nama_bidang, String nama_partai)
    {
        this.id = id;
        this.nama = nama;
        this.nama_bidang = nama_bidang;
        this.nama_partai = nama_partai;
    }

    public String get_id()
    {
        return this.id;
    }

    public void set_id(String id)
    {
        this.id = id;
    }

    public String get_nama()
    {
        return this.nama;
    }

    public void set_nama(String nama)
    {
        this.nama = nama;
    }

    public String get_nama_bidang()
    {
        return this.nama_bidang;
    }

    public void set_nama_bidang(String nama_bidang)
    {
        this.nama_bidang = nama_bidang;
    }

    public String get_nama_partai()
    {
        return this.nama_partai;
    }

    public void set_nama_partai(String nama_partai)
    {
        this.nama_partai = nama_partai;
    }
}